config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -1175,6 +1177,11 @@ protected void configure(ServerHttpSecurity http) {
 			authenticationFilter.setRequiresAuthenticationMatcher(getAuthenticationMatcher());
 			authenticationFilter.setServerAuthenticationConverter(getAuthenticationConverter(clientRegistrationRepository));
 
+			if (this.authenticationSuccessHandler == null) {
```

Comment:
Can we move this to `getAuthenticationSuccessHandler()` similar to `getAuthenticationFailureHandler()`

Reply:
Completed